assets/js/search.js

```diff
@@ -1,6 +1,5 @@
 // methods for the search page
-import { locateMe, setDateToday, preciseLocation } from "./utils.js";
-
+import { locateMe, preciseLocation, setDateToday } from "./utils.js";
```

Comment:
NIT: curious why the order of two functions are swapped? Is it done by the code formatter?

Reply:
Yes, sorted alphabetically